src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -722,6 +722,7 @@
         label = "Date of submission",
         access = {DefaultAccess.class}
     )
+    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
```

Comment:
do we need milliseconds ?

Reply:
It's what CCD expects: https://tools.hmcts.net/confluence/pages/viewpage.action?spaceKey=RCCD&title=CCD+Supported+Field+Types&__ncforminfo=mand5-SxRtTmsp4kjkR-uw0VanXD-JnuZMs1BZeym6mK0V1-26DCB4-gvqUWF2y0iA6bKT1xLf13fhTyzs_nmw==#CCDSupportedFieldTypes-ccdTypeDateTime